.circleci/config.yml

```diff
@@ -97,7 +97,7 @@ workflows:
             branches:
               ignore: /.*/
             tags:
-              only: /^\d+\.\d+\.\d+$/
+              only: /^\d+\.\d+\.\d+((a|b|rc)\d)?$/
```

Comment:
can you clarify some examples of tags that this will match?

Reply:
Sounds good! is the shorthand the normal of releasing pre release versions in python? e.g. 1.2.3.a1 instead of `1.2.3-alpha`?